### DISCUSSION TITLE
请大佬帮忙看看这是什么意思www

### DISCUSSION THREAD

**Discussion by Daydreamer114** [babalae/better-genshin-impact#202] [Q&A]
打开bettergl直接弹出p1，点启动弹出p2
![截屏2024-01-09 18 01 59](https://github.com/babalae/better-genshin-impact/assets/74444214/368f36ff-8dbe-4e61-b21d-a36dd347467c)
![截屏2024-01-09 18 05 02](https://github.com/babalae/better-genshin-impact/assets/74444214/4ce55731-cfd3-436e-91e8-5cd680f89c44)

**Comment by Daydreamer114** [comment]
环境是m2 max上的parallels上的win11，除了三月七，没自己调过东西

**Comment by huiyadanli** [comment]
https://bgi.huiyadan.com/faq.html#%E2%9D%93%E6%8F%90%E7%A4%BA-paddleocr%E9%A2%84%E7%83%AD%E5%A4%B1%E8%B4%A5-%E5%BA%94%E8%AF%A5%E5%A6%82%E4%BD%95%E8%A7%A3%E5%86%B3

看下这里的方案能否解决问题
